### PR TITLE
Release v0.70.0

### PR DESCRIPTION
**Pull request by VineethReddy02**
resolves: #1391

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 Changes by Version
 ==================
+0.70.0
+------------------
+### 💡 Enhancements 💡
+* `instrumentation/java`: Update the javaagent version to 1.23.0.([#1474](https://github.com/open-telemetry/opentelemetry-operator/pull/1474)
+#### OpenTelemetry Collector and Contrib
+* [OpenTelemetry Collector - v0.70.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.70.0)
+* [OpenTelemetry Contrib - v0.70.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.70.0)
```
Comment (frzifus) [inline CHANGELOG.md:9]: Looks like there are a few entries from [.chloggen](https://github.com/open-telemetry/opentelemetry-operator/tree/1c9610ce5dc78643f72d46e864848b45b9f48888/.chloggen) missing.

**Review by pavolloffay — COMMENTED** [review]
* the java auto instrumentation needs to be updated in versions.txt
* The changelog should be generated with make target e.g. `make chlog-preview `.
* I would like to as well adapt this PR and list all components https://github.com/open-telemetry/opentelemetry-operator/pull/1484/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR20

**Review by pavolloffay — APPROVED** [review]

**Review by pavolloffay — CHANGES_REQUESTED** [review]
Actually the release schedule table in release.md needs to be updated -

**Review by pavolloffay — APPROVED** [review]

**Review by pavolloffay — CHANGES_REQUESTED** [review]
Actually the changelog is mising now :/ 

Please include there https://github.com/open-telemetry/opentelemetry-operator/pull/1487/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR4 as a first line

**Review by pavolloffay — APPROVED** [review]

**Comment by pavolloffay** [comment]
most of the changelog files should be removed https://github.com/open-telemetry/opentelemetry-operator/tree/main/.chloggen

**Review by pavolloffay — CHANGES_REQUESTED** [review]
The changelog dir was not cleared

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,28 @@ Changes by Version
 
 <!-- next version -->
 
+0.70.0
```
Comment (pavolloffay) [inline CHANGELOG.md:6]: this should be a title

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**